roles/system/tasks/subtasks/mounts.yml

```diff
@@ -23,9 +23,14 @@
     - "{{ ansible_mounts }}"
   when: (item.mount == '/') and (item.fstype == 'ext4')
 
-- name: Mounts | Install 'fuse'
+- name: Mounts | Install 'fuse3'
```

Comment:
How sure are you that this won't break older applications?

Reply:
https://github.com/rclone/rclone/issues/3562 comes to mind

Reply:
This will definitely break `rclone cmount` since that can only work with fuse2.

Reply:
i actually didn't know this

> This will definitely break `rclone cmount` since that can only work with fuse2.

i actually didn't know this, although fuse-overlay should work with fuse2 , read somewhere never tested myself. 

but fuse3 does work in my setup, but i never checked cmount just the default mount.